.superpowers/brainstorm/95495-1779451406/content/poll-card-layouts.html

```
@@ -0,0 +1,279 @@
+<h2>Which Poll Card layout fits best?</h2>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)